### PR TITLE
install pytest, ipdb, flake8-config

### PR DESCRIPTION
**Pull request by noobOriented**

### CODE REVIEW

**Comment by SoluMilken** [comment]
要 pipenv 嗎 ????

**Comment by SoluMilken** [comment]
要邪惡的codecov嗎？？

**Comment by noobOriented** [comment]
codecov 可以自己定就好嗎，不要用那個相對的

**Comment by SoluMilken** [comment]
make file 要改 阿阿阿

**Comment by SoluMilken** [comment]
記得rebase master

**Comment by SoluMilken** [comment]
requirements.txt 可以拿掉惹 如果你要用pipenv
等等 ci config 要改一下 可以參考 https://github.com/Yoctol/uttut/blob/master/.travis.yml

File: Pipfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+[[source]]
```
Comment (SoluMilken) [inline Pipfile:1]: TODO:
- [ ] rebase master
- [ ] remove requirements.txt
- [ ] modify Makefile
- [ ] modify .travis.yml (ref: https://github.com/Yoctol/uttut/blob/master/.travis.yml)

**Review by SoluMilken — CHANGES_REQUESTED** [review]

File: Pipfile [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+verify_ssl = true
+url = "https://pypi.python.org/simple"
+name = "pypi"
+
```
Comment (SoluMilken) [inline Pipfile:5]:
```suggestion

[packages]
yoctol-argparse = {editable = true, path = "."}

```

Comment (SoluMilken) [reply Pipfile:5]: `yoctol-argparse = {editable = true, path = "."}`
 = `pip install -e .`

File: Makefile [hunk 3]
```diff
@@ -2,17 +2,19 @@
 
 .PHONY: install
 install:
-	pip install -U pip wheel setuptools
-	pip install -r requirements.txt
-	pip install -e .
+	pipenv install
+
+.PHONY: install-dev
+install:
```
Comment (SoluMilken) [inline Makefile]:
```suggestion
install-dev:
```

**Comment by SoluMilken** [comment]
加個.coveragerc 吧...
omit 一些 巴巴 
(ref: https://github.com/Yoctol/nlu-model/blob/master/.coveragerc)
```
----------- coverage: platform linux, python 3.6.3-final-0 -----------
Name                                 Stmts   Miss  Cover   Missing
------------------------------------------------------------------
setup.py                                 8      8     0%   1-14
yoctol_argparse/__init__.py              0      0   100%
yoctol_argparse/tests/__init__.py        0      0   100%
yoctol_argparse/tests/test_fake.py       2      0   100%
------------------------------------------------------------------
TOTAL                                   10      8    20%
FAIL Required test coverage of 80% not reached. Total coverage: 20.00%
```

File: Makefile [hunk 4]
```diff
@@ -2,17 +2,19 @@
...
+	pipenv install --dev
 
 .PHONY: lint
 lint:
 	flake8
 
 .PHONY: test
 test:
-	pytest
+	pytest --cov-report=term-missing --cov=. --cov-fail-under=80
```
Comment (SoluMilken) [inline Makefile]:
```suggestion
	pytest --cov-report=term-missing --cov=yoctol-argparse/ --cov-fail-under=80
```

File: Makefile [hunk 5]
```diff
@@ -2,17 +2,19 @@
...
+	pipenv install --dev
 
 .PHONY: lint
 lint:
 	flake8
 
 .PHONY: test
 test:
-	pytest
+	pytest --cov-report=term-missing --cov=yoctol-argparse/ --cov-fail-under=80
```
Comment (SoluMilken) [inline Makefile]:
```suggestion
	pytest --cov-report=term-missing --cov=yoctol_argparse/ --cov-fail-under=80
```

File: Pipfile (hunk 2 above)
Comment (SoluMilken) [reply Pipfile:5]: @noobOriented 這個

**Review by SoluMilken — APPROVED** [review]

**Pull request merged**